reactjs-todo/playwright.config.js

```diff
@@ -0,0 +1,68 @@
+import { defineConfig, devices } from '@playwright/test';
+
+const url = process.env.PLAYWRIGHT_TEST_BASE_URL || 'https://localhost:8443';
+
+export default defineConfig({
+  testDir: 'e2e',
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: 'html',
+  use: {
+    trace: 'on',
+    baseURL: `${url}`,
+    ignoreHTTPSErrors: true,
+    headless: !!process.env.CI,
+  },
+  webServer: [
+    {
+      command: 'npm start -w reactjs-todo',
+      url,
+      timeout: 120 * 1000,
+      reuseExistingServer: true,
+      cwd: '../',
+      env: {
+        API_URL: 'http://localhost:9443',
+        DEBUGGER_OFF: 'true',
+        DEVELOPMENT: 'false',
+        JOURNEY_LOGIN: 'Login',
+        JOURNEY_REGISTER: 'Registration',
+        PORT: '9443',
+        AM_URL: 'https://openam-sdks.forgeblocks.com/am',
+        REALM_PATH: 'alpha',
+        SCOPE: 'profile me.read email',
+        TIMEOUT: '3000',
+        WEB_OAUTH_CLIENT: 'CentralLoginOAuthClient-',
+        REST_OAUTH_CLIENT: 'RestOAuthClient',
+        REST_OAUTH_SECRET: 'newsecret121314!',
```

Comment:
this is concerning to me. I'm not sure how to approach this but I think we need to mask it in CI. let me think about this

Reply:
Yeah, this seems to be a copy of the `.env` file, but versioned in Github, which is a bad practice.